package.json

```diff
@@ -16,8 +16,8 @@
     "build-renderer": "cross-env NODE_ENV=production webpack --config ./configs/webpack.config.renderer.prod.babel.js --colors",
     "build-preload": "cross-env NODE_ENV=production webpack --config ./configs/webpack.config.preload.babel.js --colors",
     "dev": "concurrently \"cross-env START_HOT=1 yarn start-renderer-dev\" \"yarn start-preload-dev\"",
-    "dev:mainnet": "STX_NETWORK=mainnet concurrently \"cross-env START_HOT=1 STX_NETWORK=mainnet yarn start-renderer-dev\" \"yarn start-preload-dev\"",
-    "dev:testnet": "STX_NETWORK=testnet concurrently \"cross-env START_HOT=1 STX_NETWORK=testnet yarn start-renderer-dev\" \"yarn start-preload-dev\"",
+    "dev:mainnet": "cross-env STX_NETWORK=mainnet concurrently \"cross-env START_HOT=1 STX_NETWORK=mainnet yarn start-renderer-dev\" \"yarn start-preload-dev\"",
```

Comment:
![image](https://user-images.githubusercontent.com/1447546/110087678-3d2e2c00-7d94-11eb-8248-0b5625f6241e.png)

Reply:
Setting up a dev environment on Windows was one of the most painful experiences of the year